tests/cloudformation/checks/resource/aws/example_TransferServerIsPublic/FAIL.yaml

```diff
@@ -0,0 +1,10 @@
+AWSTemplateFormatVersion: "2010-09-09"
+Resources: 
+  PUBLIC:
+    Type: AWS::Transfer::Server
+    Properties: 
+      EndpointType: "PUBLIC"
+  VPCENDPOINT:
+    Type: AWS::Transfer::Server
+    Properties: 
+      EndpointType: "VPC_ENDPOINT"
```

Comment:
just a minor thing, wouldn't it be better to move this resource to the `PASS.yaml` and the `NONE` resource to the `FAIL.yaml`?